airbyte-webapp/src/components/ui/Text/Text.tsx

```diff
@@ -19,19 +21,25 @@ interface TextProps {
 
 const getTextClassNames = ({
   size,
+  color,
   centered,
   bold,
   inverseColor,
   gradient,
-}: Required<Pick<TextProps, "size" | "centered" | "bold" | "inverseColor" | "gradient">>) => {
+}: Required<Pick<TextProps, "size" | "color" | "centered" | "bold" | "inverseColor" | "gradient">>) => {
   const sizes: Record<TextSize, string> = {
     xs: styles.xs,
     sm: styles.sm,
     md: styles.md,
     lg: styles.lg,
   };
 
-  return classNames(styles.text, sizes[size], {
+  const colors: Record<TextColor, string> = {
+    darkBlue: styles.darkBlue,
+    grey: styles.grey,
+  };
```

Comment:
This and sizes should be moved outside this function.

Reply:
Sure! Will fix it!